### DISCUSSION TITLE
Feedback

### DISCUSSION THREAD

**Discussion by Richiedevop** [community/community#101891] [Other Feature Feedback, Questions, & Ideas]
@Richiedevop

**Comment by ghost** [comment]
Hey @Richiedevop I guess you requested a feedback for your github profile. So your profile lacks a README.md file which is very important and add more info and links about you in your bio. Sry if the comment was irrelevant.

**Comment by queenofcorgis** [comment]
Hi @Richiedevop , we're happy you're here! You are more likely to get a useful response if you are explicit about what your post entails, giving a few more details might help someone give you a nudge in the right direction.
